articles/azure-functions/start-stop-vms/deploy.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Deploy Start/Stop VMs v2 (preview)
-description: This article tells how to deploy the Start/Stop VMs v2 (preview) feature for your Azure VMs in your Azure subscription.
+title: Deploy Start/Stop VMs v2
```

Comment:
Do we still have v1 docs around somewhere? If not, maybe we don't need v2 in the title?